test/libgit2.jl

```diff
@@ -252,6 +253,14 @@ mktempdir() do dir
                 @test typeof(e) == LibGit2.GitError
                 @test startswith(sprint(show,e),"GitError(Code:ENOTFOUND, Class:OS, Failed to resolve path")
             end
+            path = joinpath(dir, "Example.BareTwo")
+            repo = LibGit2.init(path, true)
+            try
+                #just to see if this works
```

Comment:
Sorry, it's stupid Saturday for me - are you saying you think I need a `catch` here, @tkelman?